.github/workflows/docker.yml

```diff
@@ -39,10 +39,13 @@ jobs:
           name: Test report - Java
           path: build/reports/tests/unit-test/aggregated-results/
 
-  build_push:
+  build_test:
     needs: test
-    name: Build and push Docker image
+    name: Build/test Docker image
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        platform: [amd64 , arm64]
```

Comment:
There was a problem running this action (see https://github.com/MobilityData/gtfs-validator/actions/runs/5833255963)
I suspect it's because with the matrix we build 2 images but give them the same tag (ghcr.io/mobilitydata/gtfs-validator:latest, see line 95 below). Since it appears the _matrix_ strategy runs in parallel for the 2 platforms, this` could cause a race condition and it will fail or not depending on the time it takes to create and put the images in the local registry. 
I suggest you add a suffix to the DOCKER_TAGS variable on line 95 below. Something like:
`DOCKER_TAGS="${DOCKER_IMAGE}.${{ matrix.platform }}:latest"`
And see what happens.

Reply:
We need to run the arm64 version on an arm64 platform, or use emulation.